spinner/examples/context-and-action-and-error/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"time"
+
+	"github.com/charmbracelet/huh/spinner"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+
+	err := spinner.New().
+		Context(ctx).
+		ActionErr(func(context.Context, io.Writer) error {
+			time.Sleep(time.Minute)
```

Comment:
Even it's an example, I won't use a time. Minute here

You can use a 5s delay it would be enough.

I know this timeout won't be reached but three is a problem with a time.Sleep that will block everything. It's somehow OK for an example. But as it's an example, it should be written in a production-ready code

So here is what I'm suggesting

```suggestion
			select {
				case <-time.After(5 * time.Second):
				case <-ctx.Done():
			}
```